handler.go

```diff
@@ -402,8 +402,14 @@ func (fm *Frontman) RunOnce(input *Input, outputFile *os.File, interrupt chan st
 		fm.hostInfoSent = true
 	}
 
+	wg := new(sync.WaitGroup)
+	defer wg.Wait()
```

Comment:
I would not call `wg.Wait()` using `defer`.
I think an explicit `Wait` in the end of the function is more clear, especially if it's a operation that could halt execution.

Reply:
There is no difference between calling Wait in defer and explicitly at the end, except that in second case you can forget to add Wait when adding another return statement below.

Reply:
> There is no difference between calling Wait in defer and explicitly at the end, except that in second case you can forget to add Wait when adding another return statement below.

My argument was regarding readability not functionality. But we won't agree on a style anyway I guess. I have never seen a `defer wg.Wait()` in another codebase... so I would at least add comment that explains why the `Wait` is necessary in _every_ case. Why for example we have to wait in case an error happens